README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About me
+
+My name is **Yanru Owen WU**. Im currently a MPP second year student at Harris School of Public Policy. I worked in Japan for the past two years as a foregin civil servant. My story is also uploaded on my [school website](https://eall.columbian.gwu.edu/yanru-wu).
+
+## My Summer 
+My summer I spent my time in mainland China and Hongkong for internship and home visit. 
+-I went back to my hometown **Changsha, China** for family visit.
+-Then I went to **Shenzhen and Hongkong** for internship.
+
+## How do I look like?
+![me](Profile Owen WU.jpg)
```

Comment:
I am not sure why this does not work, probably because of the space in file name. Maybe replace space with "%".

Reply:
Totally agree, the code is correct but for some reason, it does not work. I am wondering if its because it could not pull the origin of the jpg. It does not have the https address (?)

Reply:
The syntax seems to be correct. One possible issue might be with the image file name. Try taking of the space in between or replace them with "_" and see if it works.